.github/workflows/main.yml

```diff
@@ -136,9 +136,13 @@ jobs:
       - name: install Windows dependencies
         run: |
           source "$CONDA/etc/profile.d/conda.sh"
-          conda create -n scsenv python=${{ matrix.python-version }}
-          conda activate scsenv
-          conda install -y scs lapack cvxpy -c conda-forge
+          if [ "${{ matrix.python-version }}" == "3.11" ]; then
+            conda create -y -n scsenv python=${{ matrix.python-version }} -c conda-forge
+          else
+            conda create -y -n scsenv python=${{ matrix.python-version }}
+            conda activate scsenv
```

Comment:
I note for 3.11 the env is created but not activated? I am not sure what level of support there is for  conda and Python 3.11 - maybe this is why. I can understand packages not being available.

Reply:
I copied it from #431. I'm not familiar with conda and don't know the line. But, it looks strange, so I try to update it.

Reply:
I updated to use `-c conda-forge` for `conda create` for all python versions because it is consistent with installing `scs`, `lapack`, and `cvxpy` from conda-forge.  ad22ea2